components/ficheSolution/modals/modal-save-fiche-solution-connected.tsx

```diff
@@ -0,0 +1,78 @@
+import { Select } from "@codegouvfr/react-dsfr/Select";
+
+import { updateFichesSolutionsProjetAction } from "@/actions/projets/update-fiches-solutions-projet-action";
+import { PFMV_ROUTES } from "@/helpers/routes";
+import { useProjetsStore } from "@/stores/projets/provider";
+import Button from "@codegouvfr/react-dsfr/Button";
+import Link from "next/link";
+import { ChangeEvent, useState } from "react";
+import { ModalSaveFicheSolutionProps } from "../ButtonSaveFicheSolution";
+
+export const ModalSaveFichesSolutionsConnected = ({ modal, ficheSolutionId }: ModalSaveFicheSolutionProps) => {
+  const [currentId, setCurrentId] = useState(-1);
+  const projets = useProjetsStore((state) => state.projets);
+  const addOrUpdateProjet = useProjetsStore((state) => state.addOrUpdateProjet);
+
+  const validate = async () => {
+    if (currentId > 0 && ficheSolutionId) {
+      const update = await updateFichesSolutionsProjetAction(currentId, [ficheSolutionId]);
```

Comment:
🟠 Ca ne fonctionne pas, quand je clique sur "rajouter au projet" j'ai cette erreur côté client  : 
modal-save-fiche-solution-connected.tsx:23 Uncaught (in promise) PrismaClientValidationError: 
```console
Invalid `prisma.projet.update()` invocation:
{
  where: {
    id: 80205955
  },
  data: {
    fiches_solutions_id: [
      "25"
    ]
    ~~~~~~
  },
  include: {
    collectivite: true
  }
}
```